src/assets/scripts/client/App.js

```diff
@@ -31,6 +32,11 @@ export default class App {
      * @param initialAirportToLoad {string}    ICAO id of the initial airport. may be the default or a stored airport
      */
     constructor(element, airportLoadList, initialAirportToLoad) {
+        // We need to put this in App, because it doesn't work in index.js
```

Comment:
this is still the incorrect spot.  at this point in the app, `AirportController` still hasn't been hydrated with airports.

this logic should happen somewhere around here: https://github.com/openscope/openscope/blob/096f38934a9d6798df6f1e621b5120b0901b8e2a/src/assets/scripts/client/airport/AirportController.js#L132

Reply:
@n8rzz is this already implemented?

https://github.com/openscope/openscope/blob/096f38934a9d6798df6f1e621b5120b0901b8e2a/src/assets/scripts/client/airport/AirportController.js#L139

Reply:
the logic above, yes that already exists in `develop`

hmm...

I think we have a different problem entirely here.  I think, in fact, this: https://github.com/openscope/openscope/blob/096f38934a9d6798df6f1e621b5120b0901b8e2a/src/assets/scripts/client/App.js#L88 is where are problems are coming from.

The initial issue described a browser hang, that line would do it when an airport file doesn't exist.